static/scss/top-app-bar.scss

```diff
@@ -59,52 +65,163 @@
       }
 
       &.button {
-        background: $primary;
+        @include media-breakpoint-up(md) {
+          background: $primary;
+          color: white;
+          border-radius: 3px;
+          box-shadow: inset 0 0 2px 0 rgba(255, 255, 255, 0.57);
+
+          &:hover {
+            background: black;
+          }
+        }
+      }
+    }
+  }
+
+  @include media-breakpoint-down(sm) {
```

Comment:
Can you use `&` and move the style definitions for `.navbar-collapse.collapsing, .navbar-collapse.show` (line 125) to be wrapped in this same larger `.navbar-collapse {}` selector? It's really confusing seeing `display: none;` here and not seeing any other styles in this block which change the display